content/news/playtest-20191020.md

```diff
@@ -0,0 +1,45 @@
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20191020-ragl.gif" alt="Explosive RAGL action">
+
+RAGL Season 8 has been packed with explosive action.
+</div>
+
+<hr>
+
+We have been relatively quiet about OpenRA's Tiberian Sun mod lately, but this does not mean it has been abandoned. Development has been focused mainly on resolving low-level game engine issues, which has been slow due to the complexity of the work involved and relative lack of manpower.
+
+Examples of this work can be seen in [release-20180923](/news/release-20180923/) which included renderer changes that roughly doubled TS's in-game performance, in [release-20190314](news/release-20190314) which fixed a collection of game-breaking bugs with Nod's cloak generator technology, and in the upcoming release which significantly improves deployable units and aircraft.
+
+Deployable units were a big part of Tiberian Sun, with several combat units, support units, and even production facilities (in the Firestorm expansion) all having the ability to switch between mobile and stationary modes. OpenRA's unit-deploy code was originally written for the classic Construction Yards, and was poorly suited for the more dynamic gameplay cases used in Tiberian Sun.
+
+This code has been overhauled to add missing features and additional quality of life improvements. Deploy commands can now be queued between other orders (great for managing transports!), and deployed units can be ordered to undeploy and move to another location (including into a transport, or onto a service depot) as a single action. The risk of accidentally packing your Construction Yard with an accidental misclick has been reduced by requiring the force-move hotkey to be held while issuing these orders.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20191020-tibsun.png" alt="Tiberian Sun update">
+
+Aircraft and deployable units work much better in the under-development Tiberian Sun mod.
+</div>
+
+The code defining aircraft and helicopter movement has also been rewritten, implementing many of the flight dynamics needed for Banshees, Orcas, and Carryalls. The Carryall's transport behavior has also been refined, with unit pick-up behaving much like the original game, but unit unloading now requiring an explicit order like other transports. This makes it possible to queue multiple transport runs, or to land the Carryall without releasing its cargo. Carryalls can now also be ordered to pick up deployed units, which will undeploy for pickup once the Carryall arrives.
+
+We are slowly working through our list of must-have features for a public release, but there are still many more to go. We still cannot predict a release date, but the Tiberian Sun md is developed in the open on GitHub (like everything else in OpenRA) so everybody is welcome to follow the progress and participate in this work.
```

Comment:
I wonder if we want to link the GitHub repo here (since we have the "Fork me on GitHub" badge it's not strictly necessary.